test/integration/smoke/test_deploy_vm_root_resize.py

```diff
@@ -114,7 +114,13 @@ def test_00_deploy_vm_root_resize(self):
         # 2. root disk has new size per listVolumes
         # 3. Rejects non-supported hypervisor types
         """
-        if(self.hypervisor.lower() == 'kvm'):
+        full_clone_config = Configurations.list(self.apiclient,
+                            name="vmware.create.full.clone")[0].value
+        if full_clone_config == 'false' and self.hypervisor.lower() == 'vmware':
+            self.skipTest("root disk resize is not supported "+
+                "when vmware.create.full.clone is %s" % full_clone_config)
+
```

Comment:
@priyankparihar @cloudsadhu test_rootvolume_resize.py test looks good. Can we add exactly the same logic of adding full clone config for smoke test test_deploy_vm_root_resize.py so that it is not skipped if a global setting is to use link clones?